tasks/queue-granules/index.js

```diff
@@ -64,6 +65,7 @@ async function queueGranules(event) {
       });
       if (executionArn) {
         const queuedGranule = {
+          collectionId: constructCollectionId(granule.dataType, granule.version),
```

Comment:
Fix for the issue where `updateGranule` cannot do a PG write of a non-existent granule without a `collectionId`